changelog/2022-04-19.md

```diff
@@ -0,0 +1,32 @@
+## :wrench: April 19, 2022
+
+**CLI version: 0.8.30**
+
+### Datastore Enhancements
+
+#### :star: Faster S3 Datastores!
+
+We are happy to announce that starting today, creating datastores from S3 buckets is near instantaneous! 
+
+In certain cases, your S3 bucket may fit one (or both) of the following criteria:
+1. the bucket is continually updating with new data which you want included in a Grid datastore 
+2. the bucket is particularly large (leading to long Datastore creation times). 
+
+In these cases, you can pass the `--no-copy` flag to the `grid datastore create` command. This flag will
+prevent grid from making a copy of the dataset, speeding up datastore creation time significantly when working with large buckets. 
+
+Example:
+`grid datastore create S3://ruff-public-sample-data/esRedditJson --no-copy` 
+
+:::note
+Please note that direct access to private S3 buckets is not currently supported.
+:::
```

Comment:
```suggestion
#### :star: Faster S3 Datastores!

We are happy to announce that, as of today, creating datastores from S3 buckets is almost instant!

In most cases, your S3 bucket will fit one (or both) of the following criteria:
1. the bucket is continually updating with new data which you want included in a Grid datastore 
2. the bucket is particularly large (leading to long datastore creation times)

In both of these cases, you can pass the `--no-copy` flag to the `grid datastore create` command. This flag will
prevent grid from making a copy of the dataset, which significantly speeds up datastore creation time when working with large buckets or ones that you want to continually update with new files.

Here's an example:
`grid datastore create S3://ruff-public-sample-data/esRedditJson --no-copy` 

:::note
Please note that direct access to private S3 buckets is not currently supported.
:::
```

Reply:
This flag will prevent grid from making a copy of the dataset, which significantly speeds up datastore creation time when working with large buckets or when making incremental updates to your bucket that do not require the entire dataset to be uploaded again. @RobertLaurella

Reply:
Two options. Square brackets are optional:

This flag will prevent Grid from making a copy of the dataset, which significantly speeds up datastore creation time when working with large buckets or if you do not want to reupload the entire dataset every time [you add a new file].

This flag will prevent Grid from making a copy of the dataset, which significantly speeds up datastore creation time when working with large buckets or when you intend to make incremental changes to your bucket and do not want to reupload the entire dataset each time [you add a new file].